src/lib/removeObjectNulls.ts

```diff
@@ -0,0 +1,5 @@
+import { curry, pickBy } from 'lodash/fp';
```

Comment:
We need to make sure that this import doesn't pull the entirety of `lodash/fp` into the client bundle. It was doing that for `lodash` imports until I added another `modularizeImports` entry to `next.config.js`.

Reply:
I recreated it using my own version of it, cough cough ChatGPT.